ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -87,12 +87,14 @@
   private final Map<String, HelixAdmin> adminForDc = new HashMap<>();
   private final HashMap<String, HashMap<DiskId, SortedSet<Replica>>> instanceToDiskReplicasMap = new HashMap<>();
   private final HashMap<String, HashMap<String, DataNodeId>> dcToInstanceNameToDataNodeId = new HashMap<>();
```

Comment:
Would be better if we change `HashMap<String, HashMap<DiskId, SortedSet<Replica>>>` to `Map<String, HashMap<DiskId, SortedSet<Replica>>>`;  

Same for `HashMap<String, HashMap<String, DataNodeId>>`

Reply:
Addressed this in the latest commit.